scripts/install/audiobookshelf.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+# Audiobookshelf installer
+
+_install_audiobookshelf() {
+    echo_progress_start "Installing audiobookshelf apt sources"
+    apt install gnupg curl
```

Comment:
We have `apt_install` defined which helps handle these without the interactivity

Reply:
You can use it like this, it is imported into the environment anytime you run `box`, you can check `sources/globals.sh` to see what else there is
```suggestion
    apt_install gnupg curl
```